tools/ldb_test.py

```
@@ -826,6 +826,68 @@ def testManifestDump(self):
+            output_lines[0],
+            r'^--------------- Column family "write"  \(ID \d+\) --------------$',
+        )
+        self.assertRegex(
+            output_lines[1],
+            r"^%d:\d+\[\d+ \.\. \d+\]"
+            r"\[.+ seq:\d+, type:\d+ \.\. .+ seq:\d+, type:\d+\]"
+            r" at level %s$" % (sst_file_number, expected_level),
+        )
+
+        self.assertRunOK("drop_column_family write", "OK")
+        output = my_check_output(
```

Ruff (0.15.21)

[error] Function call with `shell=True` parameter identified, security issue

(S604)